project/rroykuo/README.md

```diff
@@ -0,0 +1,52 @@
+# 3D-Retro-Bouncing-DVD
+
+## Basic Information
+Bouncing DVD is a retro screensaver that would appear in people's childhood. When taking tour bus and watching movie via DVD, we could see the screensaver due to device idling for a while. My object is to reproduce a DVD screensaver but in 3D space. Moreover, I'll try to make the interface to allow users create new DVD plate at any position.
```

Comment:
It is very unlikely for the code to require serious numerical calculations or sophisticated graphic.  You will probably need to provide more information about how the project demonstrates your ability to design software for numerical calculations.